.ci/docker/common/install_onnx.sh

```diff
@@ -16,7 +16,8 @@ pip_install \
   mock==5.0.1 \
   ninja==1.10.2 \
   networkx==2.5 \
-  numpy==1.24.2
+  numpy==1.24.2 \
```

Comment:
I wonder if we should upgrade numpy though.

Reply:
I guess remove numpy? It should have been installed in the main script. (Together with networkx, ninja etc)

Reply:
To be compatible with ort 1.18.1, we should keep it for now.